themes/cardinal_service/pdb_components/stories/src/app.jsx

```diff
@@ -17,6 +17,7 @@ ReactDOM.render(
     mainFiltersCount={4}
     fields={nodeFields}
     header={<h2>Search by</h2>}
+    wrapperAttributes={{className: "flex-10-of-12"}}
```

Comment:
Should we do this for resources filter too?

Reply:
Just Our Stories and Opportunities since the resources page format differs from the others (because of the secondary menu in the sidebar) 😃

Reply:
okiedokie